Unix/build.mak

```diff
@@ -392,13 +392,20 @@ CHECKDIR=/tmp/omicheck.$(shell ./buildtool username)
 check:
 	rm -rf $(CHECKDIR)
 	( cd $(CONFIGUREDIR); $(OUTPUTDIR)/install --destdir=$(CHECKDIR) )
-	(  export LD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR);  \
-           export DYLD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR); \
+	(  LD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR);  \
+	   export LD_LIBRARY_PATH;  \
+           DYLD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR); \
```

Comment:
These line up perfectly in vim.  Tabs randomizing is sort of hard to avoid in a makefile